app/templates/people/index.hbs

```diff
@@ -2,39 +2,41 @@
   <:sidebar>
     <div class="au-c-sidebar">
       <div class="au-c-sidebar__content au-o-box">
-        <form {{on "submit" this.search}} class="au-c-form">
-          <p>
+        <form {{on "submit" this.search}} class="au-o-grid au-o-grid--small">
+          <p class="au-o-grid__item">
```

Comment:
I think it might be better to show the name inputs next to each other instead of last name and organization name?